tests/end_to_end/CairoPrecompiles/test_dual_vm_token.py

```diff
@@ -146,6 +319,212 @@ async def test_should_transfer_from(
             assert balance_owner_before - amount == balance_owner_after
             assert balance_other_before + amount == balance_other_after
 
+        async def test_should_transfer_from_starknet_address_from(
```

Comment:
Factorization of tests with parametrize leads to huge decrease in readability of those tests due to little difference between them.
I suggest to let them as is